src/index.js

```diff
@@ -48,8 +51,12 @@ function getStorageObj(client, namespace) {
 
             client.hset(namespace, object.id, JSON.stringify(object), cb);
         },
+        //remove is a non-standard name, but for compatbility included here
         remove: function(id, cb) {
-            client.hdel(namespace, [id], cb);
+            return delete(id, cb);
+        },
+        delete: function(id, cb) {
```

Comment:
Would you mind covering this new method with tests? You can just copy the tests for remove and rename the describe and method call, since these should behave identically.